src/firewall/apps.py

```diff
@@ -10,34 +9,23 @@ def ready(self):
 
     def check_last_rules(self):
         from firewall.models import FirewallRule
-        rules = [
-            FirewallRule.objects.filter(
-                ip_family='IPv4', traffic_direction='Inbound', rule_id=1000),
-            FirewallRule.objects.filter(
-                ip_family='IPv4', traffic_direction='Outbound', rule_id=1000),
-            FirewallRule.objects.filter(
-                ip_family='IPv6', traffic_direction='Inbound', rule_id=1000),
-            FirewallRule.objects.filter(
-                ip_family='IPv6', traffic_direction='Outbound', rule_id=1000),
-        ]
-        
+        rules = FirewallRule.objects.filter(rule_priority=1000)
```

Comment:
the one line is equivalent to list of rules you made, it gets all rules with rule_id = 1000  (which should be 4 or zero) 
in case of db reset, these 4 rules should stay, so this function makes sure the always exist

Reply:
@dori